core/src/main/java/com/linecorp/armeria/common/util/AbstractListenable.java

```diff
@@ -17,7 +17,9 @@
 
 import static java.util.Objects.requireNonNull;
 
+import com.google.errorprone.annotations.concurrent.GuardedBy;
```

Comment:
Could you check your IDE settings and rearrange the imports?
https://armeria.dev/community/developer-guide#setting-up-your-ide

Reply:
Reordered import statements, handles ShutdownHooks files too
[8ccb411](https://github.com/line/armeria/pull/4551/commits/8ccb4115b084fbc86277f032537289d7480f2e87)

Reply:
It's not handled at all. Could you check it again, please? 😉 
The import should be:
```
import java.util.Set;
import java.util.concurrent.locks.ReentrantLock;
import java.util.function.Consumer;

import com.google.errorprone.annotations.concurrent.GuardedBy;

import com.linecorp.armeria.common.annotation.Nullable;
import com.linecorp.armeria.internal.common.util.IdentityHashStrategy;
```